views/shared/layout.jsx

```diff
@@ -11,7 +11,7 @@ module.exports = function Layout(props) {
   return (
```

Comment:
In line 28 of this file, we add a link of `buttons.css` but this file doesn't exist anymore and it causes an error in the console.

Reply:
I'll fix that up.